NodeArg class has a move constructor but doesn't have a move assignment operator

```
[ 77%] Building CXX object CMakeFiles/onnxruntime_perf_test.dir/usr/ports/math/py-onnxruntime/work-py36/onnxruntime-c7599d0/onnxruntime/test/perftest/performance_runner.cc.o
In file included from /usr/ports/math/py-onnxruntime/work-py36/onnxruntime-c7599d0/onnxruntime/test/opaque_api/test_opaque_api.cc:16:
/usr/ports/math/py-onnxruntime/work-py36/onnxruntime-c7599d0/onnxruntime/test/providers/provider_test_utils.h:440:11: error: explicitly defaulted move assignment operator is implicitly deleted [-Werror,-Wdefaulted-function-deleted]
    Data& operator=(Data&&) = default;
          ^
/usr/ports/math/py-onnxruntime/work-py36/onnxruntime-c7599d0/onnxruntime/test/providers/provider_test_utils.h:427:26: note: move assignment operator of 'Data' is implicitly deleted because field 'def_' has a deleted move assignment operator
    onnxruntime::NodeArg def_;
                         ^
```


FreeBSD 12

--- a/include/onnxruntime/core/graph/node_arg.h
+++ b/include/onnxruntime/core/graph/node_arg.h
@@ -39,6 +39,7 @@ class NodeArg {
           const ONNX_NAMESPACE::TypeProto* p_arg_type);
 
   NodeArg(NodeArg&&) = default;
+  NodeArg& operator=(NodeArg&& other) = default;
 
   /** Gets the name. */
   const std::string& Name() const noexcept;
@@ -88,8 +89,6 @@ class NodeArg {
 
   void SetType(ONNX_NAMESPACE::DataType p_type);
   void SetType(const ONNX_NAMESPACE::TypeProto& type_proto);
-
-  NodeArg& operator=(NodeArg&& other) = delete;
 
   // Node arg PType.
   ONNX_NAMESPACE::DataType type_;

